src/fetcher.ts

```diff
@@ -479,6 +480,39 @@ class HTMLHandler extends Handler {
 }
 HTMLHandler.pattern = new RegExp('text/html')
 
+class JsonLdHandler extends Handler {
```

Comment:
ok, we actually did not have a JsonLdHandler anywhere, yet? :eyes:  Thanks for writing one :+1: